tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,108 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached plot.
+
+"""
+
+###############################################################################
+# Customized Colorbars
+# ====================
+#
+# ColorbarBase derives from ScalarMappable and puts a colorbar in a specified
+# axes, so it has everything needed for a standalone colorbar. It can be used
+# as is to make a colorbar for a given colormap and does not need a mappable
+# object like an image. In this tutorial we will explore what can be done with
+# standalone colorbar.
+#
+# We will start by making a figure of desired size and adding thress axes.
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+fig, (ax1, ax2, ax3) = plt.subplots(nrows=3)
+
+###############################################################################
+# Basic continuous colorbar
+# -------------------------
+#
+# Set the colormap and norm to correspond to the data for which the colorbar
+# will be used. Then create the colorbar by calling `ColorbarBase` and
+# specify axis, colormap, norm and orientation as parameters. Here we create
+# a basic continuous colorbar with ticks and labels. There are many more kwargs
+# which can be used to further modify the colorbar.
+
+cmap = mpl.cm.cool
+norm = mpl.colors.Normalize(vmin=5, vmax=10)
+
+cb1 = mpl.colorbar.ColorbarBase(ax1, cmap=cmap,
+                                norm=norm,
+                                orientation='horizontal')
+cb1.set_label('Some Units')
+
+###############################################################################
+# Discrete intervals colorbar
+# ---------------------------
+#
+# The second example illustrates the use of a ListedColormap which generates a
+# colormap from a set of listed colors, a BoundaryNorm which generates a
+# colormap index based on discrete intervals and extended ends to show the
+# "over" and "under" value colors. Over and under are used to display data
+# outside of the normalized [0,1] range. Here we pass colors as gray shades as
+# a string encoding a float in the 0-1 range.
+#
+# If a ListedColormap is used, the length of the bounds array must be
```

Comment:
(and elsewhere in any point that we refer to a specific class / function / module in MPL)